packages/flutter/lib/src/foundation/basic_types.dart

```diff
@@ -4,10 +4,14 @@
 
 import 'dart:async';
 import 'dart:collection';
+import 'internal.dart'
+  if (dart.html) 'internal_web.dart';
 
 // COMMON SIGNATURES
 
 export 'dart:ui' show VoidCallback;
+export 'internal.dart'
```

Comment:
`internal` feel like an odd name choice here - especially for something that's exported right away?